style.css

```diff
@@ -0,0 +1,414 @@
+  width: 43px;
+  height: 43px;
+  border-radius: 50%;
+  left: 162px;
+  top: 816px;
+  display: inline-block;
+}
+
+.fb:hover {
+  background: #4267b2 url('img/fb.png') center no-repeat;
+}
+
+.block {
+  position: absolute;
+  top: 168px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  max-width: 1500px;
+}
+
+.menu__item {
+  list-style-type: none;
+  padding: 0;
+  margin: auto;
+}
+
+#menu__toggle {
+  display: none;
+}
+
+.header__burger {
+  right: 0;
+  margin-left: auto;
+  display: flex;
+  position: absolute;
+  width: 92px;
+  height: 92px;
+  background-color: #ededed;
+}
+
+.header__burger > span,
+.header__burger > span::after {
+  display: block;
+  position: absolute;
+  width: 34.7px;
+  height: 3.48px;
+  margin-left: 28px;
+  top: 39px;
+  background-color: #000;
+}
+
+.header__burger > span::after {
+  content: '';
+  margin-left: 0;
+  top: 6px;
+}
+
+  @media (min-width: 992px) {
+    .header__menu {
+      display: flex;
+      min-height: 92px;
+      min-width: 92px;
+    }
+    .header__menu:hover {
+        background: #696969;
+      }
+  }
+
+  .menu__link {
+    color: navy;
+    font-size: 25px;
+  }
+
+  .icon__line:nth-child(1) {
+    top: 0;
+  }
+
+  .icon__line:nth-child(2),
+  .icon__line:nth-child(3) {
+    top: 13px;
+  }
+
+  .icon__line:nth-child(4) {
+    top: 26px;
+  }
+
+  .menu__line:first-of-type {
+    margin-bottom: 5px;
+  }
+
+  @media (max-width: 900px) {
+    .header__menu {
+      width: 100%;
+      display: none;
+    }
+
+    .header__menu:hover {
+        background: #696969;
+      }
+
+    .header__burger {
+      width: 69px;
+      height: 69px;
+    }
+
+    .header__burger > span {
+      top: 29px;
+      margin-left: 21px;
+    }
+
+    .header__burger > span::after {
+      content: '';
+      margin-left: 0;
+    }
+  }
+
+  .new-collection {
+    position: relative;
+    display: flex;
+    min-height: 850px;
+  }
+
+  .new-collection__girl {
+    width: 549px;
+    height: 562px;
+    margin: 86px 54px 0 180px;
+  }
+
+  .new-collection__section {
+    position: relative;
+    top: 100px;
+  }
+
+  .new-collection__title {
+    width: 475px;
+    height: 164px;
+    margin-bottom: 36px;
+    left: 789px;
+    font-family: Montserrat, sans-serif;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 80px;
+    line-height: 82px;
+    color: #000;
+  }
+
+  .new-collection__description {
+    width: 508px;
+    margin-bottom: 36px;
+    font-family: Montserrat, sans-serif;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 17px;
+    line-height: 25px;
+    color: #000;
+  }
+
+  .new-collection__farther {
+    width: 203px;
+    height: 44px;
+    margin-right: 54px;
+    font-family: Montserrat, sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 0;
+    background: none;
+    text-transform: uppercase;
+    color: #000;
+    cursor: pointer;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Locc/pullRequest?prid=5986341)